rfcs/2023-09-15-endpoint-schema.md

```diff
@@ -0,0 +1,58 @@
+# Standardized Endpoint Schema
+
+| Feature name  | Example name                                |
+| :------------ | :------------------------------------------ |
+| **Author(s)** | J. Zane Cook (jzanecook@z90.studio)                      |
+| **RFC PR:**   | [PR 66](https://github.com/AI-Engineers-Foundation/agent-protocol/pull/66)                                 |
+| **Updated**   | 2023-09-15                                  |
+
+## Summary
+
+This RFC proposes several changes to the Agent Protocol API to improve task, step, and artifact management. Key changes include:
+
+- Prefixing endpoint schema with `/ap/v1/`
+- Removal of `additional_input` from tasks and steps
+- Introduction of the `/ap/v1/agent/info` endpoint
```

Comment:
remove this as well. Just keep the first one.

Reply:
https://github.com/AI-Engineers-Foundation/agent-protocol/pull/66#discussion_r1329412716